Express/controller.js

```diff
@@ -18,9 +18,14 @@ const getArticleByID = async (req, res, next) => {
   const { article_id } = req.params;
   try {
     const article = await fetchArticleByID(article_id);
-    res.status(200).send({article});
+    res.status(200).send({ article });
   } catch (err) {
     next(err);
   }
 };
-module.exports = { getApi, getTopics, getArticleByID };
+
+const getAllArticles = async (req, res, next) => {
+  const articles = await fetchArticles();
+  res.status(200).send({ articles });
+};
```

Comment:
Even though we dont expect an error to occur here, we should still wrap this in a try block with a catch block invoking next, just incase an error does happen we can capture it without breaking the app